framework/core/src/Search/SearchServiceProvider.php

```diff
@@ -30,7 +30,7 @@ public function register()
         $this->container->singleton('flarum.simple_search.fulltext_gambits', function () {
             return [
                 DiscussionSearcher::class => DiscussionFulltextGambit::class,
-                UserSearcher::class => UserFulltextGambit::class
+                UserSearcher::class => UserFulltextGambit::class,
```

Comment:
nitpick: just to keep the git history clean lol
```suggestion
                UserSearcher::class => UserFulltextGambit::class
```